src/app/templates/app/submit_new_license.html

```diff
@@ -0,0 +1,112 @@
+      <label class="control-label col-sm-3" >Full Name</label>
+        <div class="col-sm-3"> 
+          <input type="text" id="fullname" placeholder="" name = "fullname" class="alignedField">
+          <label class="control-label" id="fullnamelabel" ></label>
+        </div>
+      </div>  
+    </div>
+
+    <div class = "form-group">
+      <div class="col-sm-12">
+      <label class="control-label col-sm-3" >Short Identifier</label>
+        <div class="col-sm-3"> 
+          <input type="text" id="shortidentifier" placeholder="" name = "shortidentifier" class="alignedField" >
+          <label class="control-label" id="shortidentifierlabel" ></label>
+        </div>
+      </div>  
+    </div>
+
+    <div class = "form-group">
+      <div class="col-sm-12">
+      <label class="control-label col-sm-3" >Source / URL</label>
+        <div class="col-sm-3" >
+          <button class="roundedButton btn btn-md btn-info" id="addSourceUrl" type="submit" >+</button>
+          <input type="text" id="sourceurl" placeholder="" name = "sourceurl">
+          <label class="control-label" id="sourceurllabel" ></label>
+        </div>
+      </div>  
+    </div>
+
+    <div class = "form-group">
+      <div class="col-sm-12">  
+      <label class="control-label col-sm-3" >&nbsp;OSI Approved&nbsp;</label>
+        <div class="col-sm-3">
+           <select name="osi_approved" id="osi_approved" class="alignedField">
+              <option value="0" name ="here" selected="">-</option>
+              <option value="Yes">Yes</option>
+              <option value="No">No</option>
+          </select>
+        </div>
+      </div>
+    </div>
+
+    <div class = "form-group">
+      <div class="col-sm-12">
+      <label class="control-label col-sm-3" >Notes</label>
+        <div class="col-sm-3"> 
+          <input type="text" id="notes" placeholder="" name = "notes" class="alignedField">
+          <label class="control-label" id="noteslabel" ></label>
+        </div>
+      </div>  
+    </div>
+
+    <div class = "form-group">
+      <div class="col-sm-12">
+      <label class="control-label col-sm-3" >Standard License Header</label>
+        <div class="col-sm-3">           
+          <textarea form ="newlicense" name="licenseheader" id="licenseheader" cols="40" wrap="soft" rows="2"></textarea>
+          <label class="control-label" id="licenseheaderlabel" ></label>
+        </div>
+      </div>  
+    </div>
+
+    <div class = "form-group">
+      <div class="col-sm-12">
+      <label class="control-label col-sm-3" >Text</label>
+        <div class="col-sm-3">           
+          <textarea form ="newlicense" name="text" id="text" cols="40" wrap="soft" rows="3"></textarea>
+          <label class="control-label" id="noteslabel" ></label>
+        </div>
+      </div>  
+    </div>
+
+    <hr>
+		<input type="hidden" id="cfileformat" name="submit" value="">
+		<button class=" btn btn-md btn-info" id="submitbutton" type="submit" >Submit</button>
```

Comment:
Submit button doesn't save any data. Have you not implemented the database yet?

Reply:
Make the submit button work through AJAX. You can look at other pages like validate.html  how they send requests. Also, display warnings/error if any. 
This AJAX will also be useful to send request to GitHub API.

Reply:
You can check the AJAX request implementation including the warnings I just commited @rtgdk .